lib/archethic/metrics/poller.ex

```diff
@@ -9,31 +9,24 @@ defmodule Archethic.Metrics.Poller do
 
   alias Archethic.Metrics.Collector
 
+  alias Archethic.P2P
+  alias Archethic.P2P.Node
+
   def start_link(opts \\ []) do
     options = Keyword.get(opts, :options, name: __MODULE__)
     interval = Keyword.get(opts, :interval, 5_000)
 
     GenServer.start_link(__MODULE__, [interval], options)
   end
 
-  defp default_state do
-    default_metrics = %{
-      "archethic_mining_full_transaction_validation_duration" => 0,
-      "archethic_mining_proof_of_work_duration" => 0,
-      "archethic_p2p_send_message_duration" => 0,
-      "tps" => 0
-    }
-
-    %{pid_refs: %{}, data: default_metrics}
-  end
-
   def init([interval]) do
     timer = schedule_polling(interval)
 
-    state =
-      default_state()
-      |> Map.put(:interval, interval)
-      |> Map.put(:timer, timer)
+    state = %{
```

Comment:
```
%{}
|>Map.put(..,..);
|>Map.put(...,...);

```

??

Reply:
Explicitness is better in my opinion. 
Map.put is nice when you want to update a map but to create it, this form seems better.